src/CSharp/EasyMicroservices.IdentityMicroservice.WebApi/Controllers/IdentityController.cs

```diff
@@ -1,23 +1,123 @@
-﻿using EasyMicroservices.Cores.AspCoreApi;
+﻿using Authentications.GeneratedServices;
+using EasyMicroservices.Cores.AspCoreApi;
 using EasyMicroservices.Cores.AspEntityFrameworkCoreApi;
 using EasyMicroservices.Cores.AspEntityFrameworkCoreApi.Interfaces;
 using EasyMicroservices.Cores.Database.Interfaces;
+using EasyMicroservices.IdentityMicroservice.Contracts.Common;
+using EasyMicroservices.IdentityMicroservice.Contracts.Requests;
+using EasyMicroservices.IdentityMicroservice.Contracts.Responses;
+using EasyMicroservices.IdentityMicroservice.Helpers;
+using EasyMicroservices.IdentityMicroservice.Interfaces;
 using EasyMicroservices.ServiceContracts;
 using Microsoft.AspNetCore.Mvc;
 
 namespace EasyMicroservices.IdentityMicroservice.WebApi.Controllers
 {
+
+    [ApiController]
+    [Route("api/[controller]/[action]")]
     public class IdentityController : ControllerBase
     {
+        private readonly IConfiguration _config;
+        private readonly UsersClient _userClient;
+        private readonly IJWTManager _jwtManager;
+        private readonly IdentityHelper _identityHelper;
+        private readonly IAppUnitOfWork _appUnitOfWork;
+        private readonly string _authRoot;
+
+        public IdentityController(IAppUnitOfWork appUnitOfWork)
+        {
+            _appUnitOfWork = appUnitOfWork;
+            _config = _appUnitOfWork.GetConfiguration();
+            _authRoot = _config.GetValue<string>("RootAddresses:Authentications");
+            _userClient = new(_authRoot, new System.Net.Http.HttpClient());
+            _identityHelper = _appUnitOfWork.GetIdentityHelper();
+        }
+
+        [HttpPost]
+        public async Task<MessageContract<bool>> VerifyUserName(VerifyEmailAddressContract request)
```

Comment:
Please add VerifyUserNameResponseContract.
Why VerifyUserName has VerifyEmailAddressContract for request?